src/testing/jest/jest-27-and-under/jest-config.ts

```diff
@@ -145,7 +146,7 @@ export function buildJestConfig(config: d.ValidatedConfig): string {
     jestConfig.verbose = stencilConfigTesting.verbose;
   }
 
-  jestConfig.testRunner = 'jest-jasmine2';
+  jestConfig.testRunner = getDefaultJestRunner();
```

Comment:
Should this pull directly from the the v27 adapter rather than the abstract one? I know it evaluates to the same thing, but that would keep the version-specific adapters from "knowing about" the more generic stuff

Reply:
I'm not sure - my only concern here is that the general version has a cache of the adapter, whereas the version specific versions don't. Let me verify this won't run >1 for `npx stencil test` and I'll circle back

Reply:
Mkay - we can do this 👍 Verified that this isn't called >1, so we don't end up instantiating a new Facade instance each time we test. Updated in 615402cfec0416af55214d8af7c4a8d0599ba37d